test/pkg/environment/aws/metrics.go

```diff
@@ -50,17 +50,19 @@ const (
 	TestSubEventTypeDimension   = "subEventType"
 	TestGroupDimension          = "group"
 	TestNameDimension           = "name"
+	CommitHashDimension         = "commitHash"
 )
 
 // MeasureDurationFor observes the duration between the beginning of the function f() and the end of the function f()
-func (env *Environment) MeasureDurationFor(f func(), eventType EventType, group, name string, additionalLabels ...map[string]string) {
+func (env *Environment) MeasureDurationFor(f func(), eventType EventType, group, name, commit string, additionalLabels ...map[string]string) {
```

Comment:
Rather than passing it through to this function, can this just be a global that the test writer doesn't have to reason about? As-in, you just get this for free when you run the suite and use that function? You could event put it into `ExpectEventDurationMetric` function directly so it's completely abstracted away.